acceptance/build_info_test.go

```diff
@@ -31,6 +31,7 @@ func TestBuildInfo(t *testing.T) {
 	l := localcluster.Create(1, stopper)
 	l.Start()
 	defer l.Stop()
+	defer assertNoEvents(t, l)
```

Comment:
some tests actually do restart nodes (I'm assuming some more will soon). Really what we'd need is to process the Events channel inside of the `*Cluster` so that it can link intentional kills/restarts to their emitted events. After that, we should do what you suggest and fail on `Stop()` (which is what I initially wanted).